Index: ork/qt3d/src/3dparty/assimp/contrib/zip/src/miniz.h
===================================================================
--- ork/qt3d/src/3rdparty/assimp/contrib/zip/src/miniz.h
+++ ork/qt3d/src/3rdparty/assimp/contrib/zip/src/miniz.h
@@ -161,6 +161,10 @@
 #define MINIZ_HEADER_INCLUDED
 
 #include <stdlib.h>
+#ifndef stat64
+#include <sys/stat.h>
+#define stat64 stat
+#endif
 
 // Defines to completely disable specific portions of miniz.c:
 // If all macros here are defined the only functionality remaining will be CRC-32, adler-32, tinfl, and tdefl.
