From b88f0296c6bc644e00d2055ca2f4e53b2ef1bfad Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Thu, 27 Aug 2020 09:45:25 +0200
Subject: [PATCH 3/4] qtwebengine: Allow Windows SDK 10.0.17763

---
 configure.pri | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/qtwebengine/src/configure.pri b/qtwebengine/src/configure.pri
index d3ba9b14..a075c255 100644
--- a/qtwebengine/src/configure.pri
+++ b/qtwebengine/src/configure.pri
@@ -430,8 +430,13 @@ defineTest(qtwebengine_isWindowsPlatformSupported) {
         qtwebengine_platformError("requires MSVC or Clang (MSVC mode).")
         return(false)
     }
+<<<<<<< HEAD
     !qtwebengine_isMinWinSDKVersion(10, 18362): {
         qtwebengine_platformError("requires a Windows SDK version 10.0.18362 or newer.")
+=======
+    !qtwebengine_isMinWinSDKVersion(10, 17763): {
+        qtwebengine_platformError("Qt WebEngine on Windows requires a Windows SDK version 10.0.17763 or newer.")
+>>>>>>> qtwebengine: Allow Windows SDK 10.0.17763
         return(false)
     }
     return(true)
-- 
2.23.0

