From 0893e9b9b117a8b01b11c1df3cc2ad151e3ff778 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 12 Jul 2020 11:16:36 +0200
Subject: [PATCH 07/13] qtwebenginec: Use conda_prefix for ui_gl

---
 chromium/ui/gl/BUILD.gn | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/qtwebengine/src/3rdparty/chromium/ui/gl/BUILD.gn b/qtwebengine/src/3rdparty/chromium/ui/gl/BUILD.gn
index 5348fd57e2b..dbec6183f94 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/gl/BUILD.gn
+++ b/qtwebengine/src/3rdparty/chromium/ui/gl/BUILD.gn
@@ -36,6 +36,8 @@ config("gl_config") {
   if (use_egl) {
     defines += [ "USE_EGL" ]
   }
+  include_dirs = [ conda_prefix+"/include" ]
+  lib_dirs = [ conda_prefix+"/lib" ]
 }
 
 jumbo_component("gl") {
-- 
2.24.3 (Apple Git-128)

