From 7c6a06fb2e902a5eca6803f54e35be3250decbdf Mon Sep 17 00:00:00 2001
From: Paul Yim <paul.j.yim@gmail.com>
Date: Thu, 25 Mar 2021 09:45:16 -0400
Subject: [PATCH] Correct int types.


--- a/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctint-neon.c
+++ b/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctint-neon.c
@@ -214,13 +214,13 @@ void jsimd_idct_islow_neon(void *dct_table,
   int16x4_t bitmap = vorr_s16(row7, row6);
   bitmap = vorr_s16(bitmap, row5);
   bitmap = vorr_s16(bitmap, row4);
-  int64_t bitmap_rows_4567 = vreinterpret_s64_s16(bitmap);
+  int64_t bitmap_rows_4567 = vget_lane_s64(vreinterpret_s64_s16(bitmap), 0);
 
   if (bitmap_rows_4567 == 0) {
     bitmap = vorr_s16(bitmap, row3);
     bitmap = vorr_s16(bitmap, row2);
     bitmap = vorr_s16(bitmap, row1);
-    int64_t left_ac_bitmap = vreinterpret_s64_s16(bitmap);
+    int64_t left_ac_bitmap = vget_lane_s64(vreinterpret_s64_s16(bitmap), 0);
 
     if (left_ac_bitmap == 0) {
       int16x4_t dcval = vshl_n_s16(vmul_s16(row0, quant_row0), PASS1_BITS);
@@ -266,18 +266,18 @@ void jsimd_idct_islow_neon(void *dct_table,
   bitmap = vorr_s16(row7, row6);
   bitmap = vorr_s16(bitmap, row5);
   bitmap = vorr_s16(bitmap, row4);
-  bitmap_rows_4567 = vreinterpret_s64_s16(bitmap);
+  bitmap_rows_4567 = vget_lane_s64(vreinterpret_s64_s16(bitmap), 0);
   bitmap = vorr_s16(bitmap, row3);
   bitmap = vorr_s16(bitmap, row2);
   bitmap = vorr_s16(bitmap, row1);
-  int64_t right_ac_bitmap = vreinterpret_s64_s16(bitmap);
+  int64_t right_ac_bitmap = vget_lane_s64(vreinterpret_s64_s16(bitmap), 0);
 
   /* Initialise to non-zero value: defaults to regular second pass. */
   int64_t right_ac_dc_bitmap = 1;
 
   if (right_ac_bitmap == 0) {
     bitmap = vorr_s16(bitmap, row0);
-    right_ac_dc_bitmap = vreinterpret_s64_s16(bitmap);
+    right_ac_dc_bitmap = vget_lane_s64(vreinterpret_s64_s16(bitmap), 0);
 
     if (right_ac_dc_bitmap != 0) {
       int16x4_t dcval = vshl_n_s16(vmul_s16(row0, quant_row0), PASS1_BITS);
--- qt-everywhere-src-5.15.0.orig/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctred-neon.c
+++ qt-everywhere-src-5.15.0/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctred-neon.c
@@ -217,8 +217,8 @@ void jsimd_idct_4x4_neon(void *dct_table,
   bitmap = vorrq_s16(bitmap, row6);
   bitmap = vorrq_s16(bitmap, row7);
 
-  int64_t left_ac_bitmap = vreinterpret_s64_s16(vget_low_s16(bitmap));
-  int64_t right_ac_bitmap = vreinterpret_s64_s16(vget_high_s16(bitmap));
+  int64_t left_ac_bitmap = vget_lane_s64(vreinterpret_s64_s16(vget_low_s16(bitmap)), 0);
+  int64_t right_ac_bitmap = vget_lane_s64(vreinterpret_s64_s16(vget_high_s16(bitmap)), 0);
 
   /* Load constants for IDCT computation. */
   const int16x4x3_t consts = vld1_s16_x3(jsimd_idct_4x4_neon_consts);
--- a/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctred-neon.c
+++ b/qtwebengine/src/3rdparty/chromium/third_party/libjpeg_turbo/simd/arm/common/jidctred-neon.c
@@ -452,7 +452,8 @@ void jsimd_idct_4x4_neon(void *dct_table,
   /* Interleaving store completes the transpose. */
   uint8x8x2_t output_0123 = vzip_u8(vqmovun_s16(output_cols_02),
                                     vqmovun_s16(output_cols_13));
-  uint16x4x2_t output_01_23 = { output_0123.val[0], output_0123.val[1] };
+  uint16x4x2_t output_01_23 = { vreinterpret_u16_u8(output_0123.val[0]),
+                                vreinterpret_u16_u8(output_0123.val[1]) };

   /* Store 4x4 block to memory. */
   vst2_lane_u16((uint16_t *)(output_buf[0] + output_col), output_01_23, 0); Load constants for IDCT computation. */
